caffe2/CMakeLists.txt

```diff
@@ -1068,15 +1068,14 @@ if(USE_XPU)
   if(NOT TARGET torch_xpu_ops)
     message(WARNING "Failed to include ATen XPU implementation target")
   else()
-    target_link_libraries(torch_xpu PRIVATE torch_xpu_ops)
     if(MSVC)
       # Windows
       target_link_libraries(torch_xpu PRIVATE
       "-WHOLEARCHIVE:\"$<TARGET_FILE:torch_xpu_ops>\"")
```

Comment:
Please use cmake target for windows as well to decide dependency between `torch_xpu` and `torch_xpu_ops`.